simple_history/tests/tests/test_manager.py

```diff
@@ -72,17 +76,71 @@ class AsOfAdditionalTestCase(TestCase):
 
     def test_create_and_delete(self):
         now = datetime.now()
-        document = models.Document.objects.create()
+        document = Document.objects.create()
         document.delete()
-        for doc_change in models.Document.history.all():
+        for doc_change in Document.history.all():
             doc_change.history_date = now
             doc_change.save()
-        docs_as_of_tmw = models.Document.history.as_of(now + timedelta(days=1))
+        docs_as_of_tmw = Document.history.as_of(now + timedelta(days=1))
         self.assertFalse(list(docs_as_of_tmw))
 
     def test_multiple(self):
-        document1 = models.Document.objects.create()
-        document2 = models.Document.objects.create()
-        historical = models.Document.history.as_of(
+        document1 = Document.objects.create()
+        document2 = Document.objects.create()
+        historical = Document.history.as_of(
             datetime.now() + timedelta(days=1))
         self.assertEqual(list(historical), [document1, document2])
+
+
+class BulkHistoryCreateTestCase(TestCase):
+    def setUp(self):
+        self.data = [
+            Poll(id=1, question='Question 1', pub_date=datetime.now()),
+            Poll(id=2, question='Question 2', pub_date=datetime.now()),
+            Poll(id=3, question='Question 3', pub_date=datetime.now()),
+            Poll(id=4, question='Question 4', pub_date=datetime.now()),
+        ]
+
+    def test_simple_bulk_history_create(self):
+        created = Poll.history.bulk_history_create(self.data)
+        self.assertEqual(len(created), 4)
```

Comment:
`self.assertEqual(len(created), len(self.data))` would be less brittle in the event that we ever change the number of items in there.

Reply:
Was following how django tests their bulk_create https://github.com/django/django/blob/master/tests/bulk_create/tests.py#L28

And also prefer to have constants in the expected value